Index.html

```diff
@@ -8,60 +8,63 @@
     <title>Renome restaurant</title>
   </head>
 
-  <main>
-    <body>
-      <div class="navigation">
-        <p class="navigation-left">Renome</p>
+  <body>
+    <header>
+      <nav class="navigation">
+        <p class="navigation-logo">Renome</p>
         <div class="navigation-right">
           <img
-            class="navigation-right-cart"
+            class="navigation-right_cart"
             src="/images/cart-shopping-solid.svg"
-            alt="cart-shopping-solid"
+            alt="shopping cart icon"
           />
-          <p class="navigation-count">2</p>
+          <p class="navigation-right_count">2</p>
           <img
-            class="navigation-divide"
+            class="navigation-right_divide"
             src="/images/9034432_divider_vertical_icon.svg"
+            alt="divider icon"
           />
           <img
-            class="naviagtion-right-burger"
+            class="naviagtion-right_burger"
             src="/images/6351905_burger_checklist_list_menu_navigation_icon.svg"
-            alt="burger menu"
+            alt="burger menu icon"
           />
         </div>
-      </div>
+      </nav>
+    </header>
 
-      <div class="carousel-container">
+    <main>
+      <div class="carousel">
         <div class="carousel-images">
           <img
             src="/images/carousel_img_1.png"
             alt="peper salt timian and knife"
           />
-          <div class="carousel-container-text">
-            <h4 class="carousel-title">made with love</h4>
-            <p class="carousel-subtitle">for you</p>
-            <button class="left"><a class="arrow back">&#8249;</a></button>
-            <button class="right"><a class="arrow back">&#8250;</a></button>
-          </div>
+        </div>
+        <div class="carousel-text">
+          <h4 class="carousel-title">made with love</h4>
+          <p class="carousel-subtitle">for you</p>
+          <button class="carousel-arrow">
+            <img src="/images/left_arrow_icon.svg" alt="arrow left" />
```

Comment:
Be consistent with how write attributes, before you had for image src and alt in new lines

Reply:
Thank you, yes I'm trying to write attribute in new line so it's easier to read.